src/Bedrock.Framework.Experimental/Protocols/Memcached/MemcachedResponse.cs

```diff
@@ -0,0 +1,57 @@
+﻿using Bedrock.Framework.Infrastructure;
+using System;
+using System.Buffers;
+using System.Buffers.Binary;
+using System.Collections.Generic;
+using System.Text;
+using static Bedrock.Framework.Experimental.Protocols.Memcached.Enums;
+
+namespace Bedrock.Framework.Experimental.Protocols.Memcached
+{
+    public class MemcachedResponse
+    {
+        public ReadOnlyMemory<byte> Data { get; private set; }
```

Comment:
This should be a `ReadOnlySequence<byte>`.

Reply:
Fixed